python/ray/serve/tests/test_model_wrappers.py

```diff
@@ -72,6 +74,35 @@ def test_batching(serve_instance):
         assert resp == {"value": [42], "batch_size": 2}
 
 
+def test_model_wrappers_in_pipeline(serve_instance):
+    _, path = tempfile.mkstemp()
+    with open(path, "w") as f:
+        json.dump(2, f)
+
+    predictor_cls = "ray.serve.tests.test_model_wrappers.AdderPredictor"
+    checkpoint_cls = "ray.serve.tests.test_model_wrappers.AdderCheckpoint"
+    schema_cls = "ray.serve.http_adapters.array_to_databatch"
+
+    with PipelineInputNode(preprocessor=schema_cls) as dag_input:
+        m1 = ray.remote(ModelWrapper).bind(
+            predictor_cls=predictor_cls,  # TODO: can't be the raw class right now?
+            checkpoint={  # TODO: can't be the raw object right now?
```

Comment:
I think the deeper issues is that how can I make _my own **object**__ JSON serializable so it can be picked up as part of to_yaml after build for production.

Reply:
ah, i was referring to class / func to import path. Objects can be way trickier given we enforce json .. but looks like you're using cls + path, so is it expected to let users later on to just use our built-in or custom written cls + checkpoint path instead ? I think this is one of the examples we don't want user to pass raw object .. as there'll be ppl who try to put the raw checkpoint object there :facepalm: